classroom/snippets/src/main/java/AddAliasToCourse.java

```diff
@@ -12,46 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
 // [START classroom_add_alias_to_course_class]
 
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
 import com.google.api.client.googleapis.json.GoogleJsonError;
 import com.google.api.client.googleapis.json.GoogleJsonResponseException;
-import com.google.api.client.http.HttpRequestInitializer;
 import com.google.api.client.http.javanet.NetHttpTransport;
 import com.google.api.client.json.gson.GsonFactory;
 import com.google.api.services.classroom.Classroom;
 import com.google.api.services.classroom.ClassroomScopes;
 import com.google.api.services.classroom.model.CourseAlias;
-import com.google.auth.http.HttpCredentialsAdapter;
-import com.google.auth.oauth2.GoogleCredentials;
 import java.io.IOException;
+import java.security.GeneralSecurityException;
 import java.util.Collections;
+import java.util.List;
 
 /* Class to demonstrate the use of Classroom Create Alias API. */
 public class AddAliasToCourse {
+
   /**
    * Add an alias on an existing course.
    *
    * @param courseId - id of the course to add an alias to.
    * @return - newly created course alias.
    * @throws IOException - if credentials file not found.
```

Comment:
Need an annotation for the second exception type, `@throws GeneralSecurityException`.